src/Microsoft.Health.Dicom.Api/Features/Filters/DicomRequestContextRouteDataPopulatingFilterAttribute.cs

```diff
@@ -34,9 +35,12 @@ public sealed class DicomRequestContextRouteDataPopulatingFilterAttribute : Acti
     public override void OnActionExecuting(ActionExecutingContext context)
     {
         EnsureArg.IsNotNull(context, nameof(context));
+
         IDicomRequestContext dicomRequestContext = _dicomRequestContextAccessor.RequestContext;
         dicomRequestContext.RouteName = context.ActionDescriptor?.AttributeRouteInfo?.Name;
 
+        dicomRequestContext.Version = context.HttpContext.GetRequestedApiVersion()?.MajorVersion;
```

Comment:
Could we add a test for this scenario `DicomRequestContextDataPopulatingFilterAttributeTests`

Reply:
I need to add an EnE test to validate it. I will add one E2E behind FF

Reply:
> 

URSA needs to be added after the FF is enabled